README.md

```diff
@@ -329,11 +329,12 @@ Throughout this list you'll see next to each resource and emoji. Here's what eac
 
 ## DevOps
 
+- :bulb: [Best tools for DevOps](https://kush01.hashnode.dev/best-tools-for-devops)
```

Comment:
Please move this to the end of the list (as resources should be added to the end of a category's list

Reply:
cool